buddyboss/buddyboss-platform
# Forums > Support @mentions

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #269  .

### How to test the changes in this Pull Request:

1. Go to forums -> Add new topic
2. Type topic content in the editor and type @mention_user_name
3. It should show suggestion list of users

### Proof Screenshots or Video



### Other information:




### Changelog entry

> Forums > @mentions support added.
